docs/source/abc_samplers/index.rst

```diff
@@ -13,4 +13,5 @@ given a :class:`LogPrior` and a :class:`ErrorMeasure`.
 .. toctree::
 
     base_classes
-    rejection_abc
\ No newline at end of file
+    rejection_abc
```

Comment:
I understand rejection is simpler, but these should probably be alphabetical (consistent with rest and sustainable if we have 20 abc methods)